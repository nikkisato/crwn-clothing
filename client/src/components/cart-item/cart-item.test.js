import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

it('should render CartItem component', () => {
  let wrapper;
  const mockItem = {
    imageUrl: 'www.testImage.com',
    price: 10,
    name: 'hats',
    quantity: 2,
  };
  wrapper = shallow(<CartItem item={mockItem} />);
  expect(wrapper.debug()).toMatchSnapshot();
});
