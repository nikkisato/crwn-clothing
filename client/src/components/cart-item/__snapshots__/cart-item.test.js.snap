// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartItem component 1`] = `
"<styled.div>
  <styled.img src=\\"www.testImage.com\\" alt=\\"item\\" />
  <styled.div>
    <span>
      hats
    </span>
    <span>
      2
       x $
      10
    </span>
  </styled.div>
</styled.div>"
`;
